app/src/main/java/com/example/criminalintent/presentation/crimeDetail/CrimeFragment.kt

```diff
@@ -101,23 +99,26 @@ class CrimeFragment : Fragment(), FragmentResultListener {
     }
 
     private fun addMenuItems() {
-        requireActivity().addMenuProvider(object : MenuProvider {
```

Comment:
Please do not do tasks outside of the ticket in the same PR. I thought this changes is related to dependency injection and was having a hard time identifying why.
Please create new branches and PR for other changes.

Reply:
For now, it's ok to leave it like this